src/ResourceManager/Resources/Commands.Resources/AzureRM.Resources.psd1

```diff
@@ -165,7 +169,15 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Update Roleassignment and roledefinition cmdlets:
+        ReleaseNotes = '* Support template deployment at subscription scope. Add new Cmdlets:
```

Comment:
@Tiano2017 revert the change made to the `ReleaseNotes` field here -- we will updated this section accordingly

Reply:
fixed.